Assets/MixedRealityToolkit/_Core/Devices/MixedReality/HoloLensClicker.cs

```diff
@@ -1,42 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
-using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces.Devices;
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
-using System.Collections.Generic;
-
-namespace Microsoft.MixedReality.Toolkit.Internal.Devices.WindowsMixedReality
-{
-    public struct HoloLensClicker : IMixedRealityController
```

Comment:
Do we no longer need a clicker controller class?

Reply:
We won't need it because all the WMR devices are treated the same by the Interaction Manager

Reply:
it was renamed :D

Reply:
We'll revisit when we implement the clicker